ci-operator/jobs/openshift/knative-eventing-sources/openshift-knative-eventing-sources-release-0.4-presubmits.yaml

```diff
@@ -28,8 +28,8 @@ presubmits:
         - name: CONFIG_SPEC
           valueFrom:
             configMapKeyRef:
-              key: openshift-knative-eventing-sources-master.yaml
-              name: ci-operator-master-configs
+              key: openshift-knative-eventing-sources-release-0.4.yaml
+              name: ci-operator-misc-conffigs
```

Comment:
"conffigs" ? where does it come from? the CI is complaining about it. Maybe pulling the latest image and re-generating the configs would help?